package org.mock.exception;

public class PlayerFoundException extends RuntimeException {
    public PlayerFoundException(String s) {
        super(s);
    }
}
